packages/client/src/components/atoms/AddToCalendar/__tests__/AddToCalendar.test.tsx

```diff
@@ -11,10 +11,7 @@ import * as reactRedux from "react-redux";
 import AddToCalendar from "../AddToCalendar";
 
 import { expectedIcsFile } from "../__testData__/testData";
-import {
-  __inputDialogSubmitId__,
-  __emailInput__,
-} from "@/__testData__/testIds";
+import { __inputDialogSubmitId__, __emailInput__ } from "@eisbuk/shared";
```

Comment:
I really don't like this, but I understand you didn't have much of a choice (I chose to copy paste the stuff, which is equally shitty)